dll/win32/shell32/wine/brsfolder.c

```diff
@@ -518,7 +531,12 @@ static void BrsFolder_CheckValidSelection( browse_info *info, LPTV_ITEMDATA lptv
     else
         EnableWindow(GetDlgItem(info->hWnd, IDC_BROWSE_FOR_FOLDER_NEW_FOLDER), TRUE);
 
+#ifdef __REACTOS__
+    if (bEnabled)
+            SendMessageW(info->hWnd, BFFM_ENABLEOK, 0, bEnabled);
+#else
```

Comment:
```suggestion
#endif
```

Reply:
The only ROS-specific change here being the extra `if (bEnabled)`, no need to also copy-paste the same SendMessage call, only differing by a different indentation level.